pkg/controllers/resourcechange/resourcechange_controller_test.go

```diff
@@ -671,20 +671,41 @@ func TestCollectAllAffectedPlacementsV1Beta1_ClusterResourcePlacement(t *testing
 			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "test-nameSpace",
+			Name: "test-namespace",
 			Labels: map[string]string{
 				"region":  rand.String(10),
 				"version": rand.String(4),
 			},
 		},
 	}
+
+	// Common ResourceIdentifier for Namespace tests (cluster-scoped)
```

Comment:
Where is the test case for the namespaceOnly CRP that should not match the resource?

Reply:
existing test, "Skip a placement with selecting ns only"

Reply:
I don't get it, if a test already exists then how come it passed?  I was looking for changes in the existing tests to catch the bug but all I see is just addin a key.

Reply:
The bug is in where we compare the resource with the status. Before, we used the ns so any namespace scoped resource will match with the ns only CRP. Now, i changed to use the original key for the namespace scoped resource.

These kind of tests were missing in the original UT.

Reply:
or you're talking about, adding a test case for the namespaceOnly CRP that should not match the resource in the status?